easybuild/easyconfigs/n/NLopt/NLopt-2.7.1-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'CMakeMake'
+
+name = 'NLopt'
+version = '2.7.1'
+
+homepage = 'http://ab-initio.mit.edu/wiki/index.php/NLopt'
+description = """ NLopt is a free/open-source library for nonlinear optimization,
+ providing a common interface for a number of different free optimization routines
+ available online as well as original implementations of various other algorithms. """
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/stevengj/nlopt/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = ['db88232fa5cef0ff6e39943fc63ab6074208831dc0031cf1545f6ecd31ae2a1a']
+
+builddependencies = [
+    ('CMake', '3.23.1'),
+    ('binutils', '2.34'),
+    ('SciPy-Stack', '2022a'),
+]
+multi_deps = {'Python': ['3.8', '3.9', '3.10'] }
+modextrapaths = {
+	'EBPYTHONPREFIXES': '',
+}
+
+configopts = '-DBUILD_SHARED_LIBS=ON -DNLOPT_PYTHON=ON -DPYTHON_EXECUTABLE=$EBROOTPYTHON/bin/python '
+postinstallcmds = ['/cvmfs/soft.computecanada.ca/easybuild/bin/setrpaths.sh --path %(installdir)s/lib/ --add_path %(installdir)s/lib64']
```

Comment:
`%(installdir)s/lib64` instead as `find` in `setrpaths` does not follow symlinks